docs/user-guide/petset.md

```diff
@@ -0,0 +1,278 @@
+
+* Pet Set is an *alpha* resource, not available in any Kubernetes release prior to 1.3.
+* As with all alpha/beta resources, it can be disable through the `--runtime-config` option passed to the apiserver, and in fact most likely will be disabled on hosted offerings of Kubernetes.
+* The only updatable field on a Pet Set is `replicas`
+* The storage for a given pet must either be provisioned by a [dynamic storage provisioner](http://releases.k8s.io/{{page.githubbranch}}/examples/experimental/persistent-volume-provisioning/README.md) based on the requested `storage class`, or pre-provisioned by an admin. Note that dynamic volume provisioning is also currently in alpha.
+* Deleting the Pet Set  *will not*  delete any pets. You will either have to manually scale it down to 0 pets first, or delete the pets yourself.
+* Deleting and/or scaling a Pet Set down will *not* delete the volumes associated with the Pet Set. This is done to ensure safety first, your data is more valuable than an auto purge of all related Pet Set resources. **Deleting the Persistent Volume Claims will result in a deletion of the associated volumes**.
+* All Pet Sets currently require a "governing service", or a Service responsible for the network identity of the pets. The user is responsible for this Service.
+* Updating an existing Pet Set is currently a manual process, meaning you either need to deploy a new Pet Set with the new image version, or orphan Pets one by one, update their image, and join them back to the cluster.
+
+## Example Pet Set
+
+We'll create a basic Pet Set to demonstrate how Pets are assigned unique and "sticky" identities.
+
+{% include code.html language="yaml" file="petset.yaml" ghlink="/docs/user-guide/petset.yaml" %}
+
+Saving this config into `petset.yaml` and submitting it to a Kubernetes cluster should creae the defined Pet Set and Pets it manages:
+
+```shell
+$ kubectl create -f petset.yaml
+service "nginx" created
+petset "nginx" deleted
+```
+
+## Pet Identity
+
+The identity of a Pet sticks to it, regardless of which node it's (re) scheduled on. We can examine the identity of the pets we just created.
+
+### Ordinal index
+you should see 2 pods with predictable names formatted thus: `$(petset name)-$(ordinal index assigned by petset controller)`
+
+```shell
+$ kubectl get po
+NAME      READY     STATUS    RESTARTS   AGE
+web-0     1/1       Running   0          10m
+web-1     1/1       Running   0          10m
+```
+
+### Stable storage
+2 persistent volumes, one per pod. This is auto created by the Pet Set based on the `volumeTemplate` field
+
+```shell
+$ kubectl get pv
+NAME                                       CAPACITY   ACCESSMODES   STATUS    CLAIM               REASON    AGE
+pvc-90234946-3717-11e6-a46e-42010af00002   1Gi        RWO           Bound     default/www-web-0             11m
+pvc-902733c2-3717-11e6-a46e-42010af00002   1Gi        RWO           Bound     default/www-web-1             11m
+```
+
+### Network identity
```

Comment:
Blank line btx heading and text plz.

Do this throughout the PR, just mentioning it once here.